### PR TITLE
View all exercises

### PR DESCRIPTION
**Pull request by BenjaminGrabow**
##  What does this PR do
This Pull Request sets up the Endpoint for getting all exercises.

##  Description of task to be completed
- Create findExercises in exercise-helpers.js

- Create GET endpoint for getting all exercises in exercise-routes.js

##  What are the relevant Trello board stories
- As a user I want to be able to view all the exercises:

https://trello.com/c/EBqfiwiJ/80-as-a-user-i-want-to-be-able-to-view-all-the-exercises

### CODE REVIEW

**Comment by BenjaminGrabow** [comment]
[As a user, I want to be able to view all the exercises.](https://trello.com/c/EBqfiwiJ/80-as-a-user-i-want-to-be-able-to-view-all-the-exercises)

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+routes > 
+         exercises >
+            exercise-helpers.js
+            exercise-routes.js
+            exercise-tests.spec.js
+
+         users
+         workouts
+
+Individual exercise/workout endpoints to be /exercises/:id   or /workouts/:id
```
Comment (haywhyze) [inline README.md:18]: These comments should not be on the readme

File: exercises/exercisesModels.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const db = require('../../models/dbConfig');
+
+function findExercises() {
+  return db('exercises');
+}
+
+function findExerciseById(id) {
```
Comment (haywhyze) [inline exercises/exercisesModels.js]: If the Pull Request is to view all exercises, this function meant to get one exercise by id should not be included. I think Matt is already working on this.

File: exercises/exercisesRoutes.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+router.get('/', async (req, res) => {
+  try {
+    const exercises = await db.findExercises();
+    res.status(200).json(exercises);
+  } catch (error) {
+    res.status(500).json({ Error: 'Internal Server Error' });
+  }
+});
+
+router.get('/:id', async (req, res) => {
```
Comment (haywhyze) [inline exercises/exercisesRoutes.js]: If the Pull Request is to view all exercises, this endpoint is meant to get one exercise by id should not be included. I think Matt is already working on this.

**Review by haywhyze — CHANGES_REQUESTED** [review]
This endpoint should be restricted to logged-in users. You might need to wait for @hyetigran to get his authentication ready before this task can be completed.

**Review by shaunmcarmody — APPROVED** [review]

**Pull request merged**